power_grid_model_c/power_grid_model/include/power_grid_model/common/exception.hpp

```diff
@@ -112,7 +112,12 @@ class SparseMatrixError : public PowerGridError {
 
 class NotObservableError : public PowerGridError {
   public:
-    NotObservableError() { append_msg("Not enough measurements available for state estimation.\n"); }
+    NotObservableError(std::string const& msg = "") {
```

Comment:
please also add it to the list of observability checks in https://power-grid-model.readthedocs.io/en/stable/user_manual/calculations.html#state-estimation

Reply:
It is already there. We need at least one voltage sensor.

Reply:
Based on the requirements of observability mentioned above, user needs to satisfy at least the following conditions for state estimation calculation in power-grid-model.

n_voltage_sensor >= 1